libs/permissions/api/src/lib/permission.guard.ts

```diff
@@ -18,7 +18,7 @@ export class PermissionsGuard implements CanActivate {
     ])
 
     if (!permissions.length) {
-      return true
+      return false
```

Comment:
Can revert this now I guess

Reply:
Hmm, I guess it doesnt make a difference. Basically passing empty `@Permissions()` will act as `@Public` if i return `true` here. I say that is not intended, wdyt?
I would expect if you provider `@Permissions` you wanna restrict, otherwise use `@Public` / `@Authenticated`.